Documentation/data-collection.md

```diff
@@ -21,318 +37,495 @@ data:
+    # (@openshift/openshift-team-monitoring)
+    #
+    # openshift:prometheus_tsdb_head_series:sum tracks the total number of active series
     - '{__name__="openshift:prometheus_tsdb_head_series:sum"}'
     #
-    # (monitoring-team) openshift:prometheus_tsdb_head_samples_appended_total:sum tracks the rate of samples ingested
+    # (@openshift/openshift-team-monitoring)
+    #
+    # openshift:prometheus_tsdb_head_samples_appended_total:sum tracks the rate of samples ingested
     # by prometheusi.
     - '{__name__="openshift:prometheus_tsdb_head_samples_appended_total:sum"}'
     #
-    # (monitoring-team) monitoring:container_memory_working_set_bytes:sum tracks the memory usage of the monitoring
+    # (@openshift/openshift-team-monitoring)
+    #
+    # monitoring:container_memory_working_set_bytes:sum tracks the memory usage of the monitoring
     # stack.
     - '{__name__="monitoring:container_memory_working_set_bytes:sum"}'
     #
-    # (monitoring-team) monitoring:haproxy_server_http_responses_total:sum tracks the number of times users access
+    # (@openshift/openshift-team-monitoring)
+    #
+    # monitoring:haproxy_server_http_responses_total:sum tracks the number of times users access
     # monitoring routes.
     - '{__name__="monitoring:haproxy_server_http_responses_total:sum"}'
     #
-    # (rhmi, @openshift/openshift-team-cluster-manager) rhmi_status reports the status of an RHMI installation.
+    # (https://github.com/integr8ly, @david-martin)
+    #
+    # rhmi_status reports the status of an RHMI installation.
     # Possible values are bootstrap|cloud-resources|monitoring|authentication|products|solution-explorer|deletion|complete.
     # This metric is used by OCM to detect when an RHMI installation is complete & ready to use i.e. rhmi_status{stage='complete'}
+    #
+    # consumers: (@openshift/openshift-team-cluster-manager)
     - '{__name__="rhmi_status"}'
     #
-    # (workloads-team, @openshift/openshift-group-b) cluster_legacy_scheduler_policy reports whether the scheduler operator is
+    # (openshift/openshift-team-master, @openshift/openshift-group-b)
+    #
+    # cluster_legacy_scheduler_policy reports whether the scheduler operator is
     # configured with a custom Policy file. This value is a boolean 0|1
     - '{__name__="cluster_legacy_scheduler_policy"}'
     #
-    # (workloads-team, @openshift/openshift-group-b) cluster_master_schedulable reports whether mastersSchedulable=true in
+    # (openshift/openshift-team-master, @openshift/openshift-group-b)
+    #
+    # cluster_master_schedulable reports whether mastersSchedulable=true in
     # the scheduler operator. This value is a boolean 0|1
     - '{__name__="cluster_master_schedulable"}'
     #
-    # (codeready workspaces, @ibuziuk) The number of workspaces with a given status STARTING|STOPPED|RUNNING|STOPPING. Type 'gauge'.
+    # (codeready workspaces, @ibuziuk)
```

Comment:
would be great to have a valid team name for codeready. It makes it easier to reach out the folks responsible for this metric.

Reply:
same here about teams not being part of the openshift org.